src/test/java/com/google/sps/servlets/ContestServletTest.java

```diff
@@ -0,0 +1,122 @@
+package com.google.sps.servlets;
+
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.appengine.tools.development.testing.LocalURLFetchServiceTestConfig;
+import java.io.PrintWriter;
+import java.util.HashSet;
+import java.util.Set;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.*;
```

Comment:
Don't use * per the java style guide. 

The reason why is that it makes refactoring harder, and it makes auto-import/cleanup unused import tools harder, and it makes readability harder (sometimes I want to see where a class is imported from, and * stops me from just ctrl-fing it).